admin_scripts/os2displaypc/hide_cursor_inactive.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env sh
+
+lower() {
+    echo "$@" | tr '[:upper:]' '[:lower:]'
+}
+
+ACTIVATE="$(lower "$1")"
+
+export DEBIAN_FRONTEND=noninteractive
+FILE="/home/chrome/.xinitrc"
+
+apt-get update --assume-yes
+
+if [ "$ACTIVATE" != 'false' ] && [ "$ACTIVATE" != 'falsk' ] && \
+   [ "$ACTIVATE" != 'no' ] && [ "$ACTIVATE" != 'nej' ]; then
+
+  apt-get install --assume-yes unclutter
```

Comment:
Det ser fint og effektivt ud!

Et enkelt spørgsmål: Læser man beskrivelsen af modulet unclutter f.eks. med `apt show unclutter`, ser man:

```
This is the original unclutter implementation from the 90s which
 works well with classic X11 programs. For a more modern
 implementation which works better with some more modern applications
 and UI libraries, see unclutter-xfixes.
 ```
 
 Burde vi måske bruge `unclutter-fixes` i stedet? Er der en risiko for, at Chromium er en af de "modern applications", som de skriver om?

Reply:
Det nuværende virker med Chromium, så ved ikke om det giver mening at skulle til at teste igen?
Men synes det ville være en god idé som minimum at tilføje en kommentar i scriptet om det som mulighed, i tilfælde af der senere skulle opstå problemer med det.